cli/tests/pcluster/templates/test_cluster_stack.py

```diff
@@ -767,6 +770,55 @@ def test_head_node_dna_json(mocker, test_datadir, config_file_name, expected_hea
     assert_that(generated_head_node_dna_json).is_equal_to(default_head_node_dna_json)
 
 
+@pytest.mark.parametrize(
+    "config_file_name, expected_login_node_dna_json_fields",
+    [
+        (
+            "login-basic-config.yaml",
+            {
+                "ephemeral_dir": "/scratch",
+                "enable_intel_hpc_platform": "false",
+                "head_node_private_ip": "{'Ref': 'referencetoclusternameHeadNodeENI6497A502PrimaryPrivateIpAddress'}",
+                "dns_domain": "{'Ref': 'referencetoclusternameClusterDNSDomain8D0872E1Ref'}",
+                "hosted_zone": "{'Ref': 'referencetoclusternameRoute53HostedZone2388733DRef'}",
+                "log_group_name": "/aws/parallelcluster/clustername",
+                "node_type": "LoginNode",
+                '"proxy"': "NONE",
+                "scheduler": "slurm",
+            },
+        ),
+        (
+            "login-with-directory-service.yaml",
+            {
+                "domain_read_only_user": "cn=ReadOnlyUser,ou=Users,ou=CORP,dc=corp,dc=sirena,dc=com",
+                "generate_ssh_keys_for_users": "true",
+            },
+        ),
+    ],
+)
+def test_login_node_dna_json(mocker, test_datadir, config_file_name, expected_login_node_dna_json_fields):
+    mock_aws_api(mocker)
+    mock_bucket_object_utils(mocker)
+
+    # Read yaml and render CF Template
+    input_yaml = load_yaml_dict(test_datadir / config_file_name)
```

Comment:
You haven't added the yaml file to the PR.

Reply:
OMG !
... fixing it.